testproject/subproject/build.gradle

```diff
@@ -20,3 +21,16 @@ dependencies {
     implementation(accessTransformers(project(':common')))
     accessTransformersElements(project(':common'))
 }
+
+publishing {
```

Comment:
This is good for manual testing, but I wonder if it would be possible and worthwhile to have an automated integration test as well?

Reply:
That sounds painful